bin/bails-menu

```diff
@@ -33,24 +33,26 @@ export DATA_DIR="/live/persistence/TailsData_unlocked/Persistent/.bitcoin"
 export BAILS_DIR="$LOCAL_DIR"
 
 onlynet_onion() {
-zenity --question --title='Network Privacy' --width=420 --text='Bails offers different settings for connecting to the Bitcoin network. By default, Bails connects to both clearnet and Tor (onion) peers, providing a balance between usability and privacy.
+    zenity --question --title='Network Privacy' --width=420 --text='Bails offers different settings for connecting to the Bitcoin network. By default, Bails connects to both clearnet and Tor (onion) peers, providing a balance between usability and privacy.
 
-Would you like to enhance privacy by connecting exclusively to Tor (onion) peers with reduced clearnet exposure? This may limit your network diversity and can be less stable if you have limited Tor peers.' --icon-name=tor-connection --default-cancel
+    Would you like to enhance privacy by connecting exclusively to Tor (onion) peers with reduced clearnet exposure? This may limit your network diversity and can be less stable if you have limited Tor peers.' --icon-name=tor-connection --default-cancel
 
-if [ $? = 0 ]; then
-    # User chose to enable -onlynet=onion
-    sed --in-place 's/#onlynet=<net>/onlynet=onion/' $DATA_DIR/bitcoin.conf
-    zenity --info --title='Network settings updated' --text='You have chosen to connect exclusively to Tor (onion) peers.
+    zenity_exit_status=$?
+    
+    if [ $zenity_exit_status = 0 ]; then
+        # User chose to enable -onlynet=onion
+        sed --in-place 's/#onlynet=<net>/onlynet=onion/' $DATA_DIR/bitcoin.conf
+        zenity --info --title='Network settings updated' --text='You have chosen to connect exclusively to Tor (onion) peers.
 
-Your network settings have been updated.' --width=420 --icon-name=tor-connected-symbolic
+    Your network settings have been updated.' --width=420 --icon-name=tor-connected-symbolic
 
-else
-    # User chose default network settings
-    sed --in-place 's/onlynet=onion/#onlynet=<net>/' $DATA_DIR/bitcoin.conf
-    zenity --info --title='Default network settings' --text='You have chosen to keep the default network settings.
-Bitcoin Core will continue to connect to both clearnet and Tor (onion) peers, providing a balance between usability and privacy.' --width=420 --icon-name=tor-disconnected-symbolic
+    else
+        # User chose default network settings
+        sed --in-place 's/onlynet=onion/#onlynet=<net>/' $DATA_DIR/bitcoin.conf
+        zenity --info --title='Default network settings' --text='You have chosen to keep the default network settings.
+    Bitcoin Core will continue to connect to both clearnet and Tor (onion) peers, providing a balance between usability and privacy.' --width=420 --icon-name=tor-disconnected-symbolic
```

Comment:
lets delete ", providing a balance... privacy." the sentence is ridiculous enough to say once, I don't want it repeated lol.

Reply:
Will do

Reply:
Resolved with https://github.com/epiccurious/Bails/pull/7/commits/c45a502d529c68262b0f72ce4a9463b3e6a6d7ed.